src/components/Tooltip/PopoverAnchorTooltip.tsx

```diff
@@ -10,8 +10,14 @@ function PopoverAnchorTooltip({shouldRender = true, children, ...props}: Tooltip
 
     const isPopoverRelatedToTooltipOpen = useMemo(() => {
         // eslint-disable-next-line @typescript-eslint/dot-notation
-        const tooltipNode = tooltipRef.current?.['_childNode'] ?? null;
-        if (isOpen && popover?.anchorRef?.current && tooltipNode && (tooltipNode.contains(popover.anchorRef.current) || tooltipNode === popover.anchorRef.current)) {
+        const tooltipNode: Node | null = tooltipRef.current?.['_childNode'] ?? null;
+
+        if (
+            isOpen &&
+            popover?.anchorRef?.current &&
+            tooltipNode &&
+            ((popover.anchorRef.current instanceof Node && tooltipNode.contains(popover.anchorRef.current)) || tooltipNode === popover.anchorRef.current)
```

Comment:
Let's test popovers after this change (on web/desktop):
![image](https://github.com/Expensify/App/assets/46095609/5707f13c-21b6-43e4-b0ab-fd7f78c444ef)